### PR TITLE
Fix missing tabs - Embed

### PR DESCRIPTION
**Pull request by hariombalhara**
*This should fix some of the failing tests as well*

Before:
Screenshot 2022-05-18 at 3 03 32 PM

After:
Screenshot 2022-05-18 at 3 06 18 PM

### CODE REVIEW

**Review by sean-brydon — APPROVED** [review]

File: apps/web/contexts/contractsContext.tsx
Comment (alishaz-polymath) [inline apps/web/contexts/contractsContext.tsx:42]: I'm not sure if we would address the `any` later on or not, because we generally  try to stay away from using `any` as it simply means we're ignoring typesafety in this case.

File: packages/embeds/embed-core/src/preview.ts
Comment (alishaz-polymath) [inline packages/embeds/embed-core/src/preview.ts:11]: same as above regarding the use of `any`

**Review by alishaz-polymath — CHANGES_REQUESTED** [review]
Looks good overall, just a little concern of using `any` type as I remember @zomars working on getting rid of all `any` instances a while back. So, I'd advise to wait for @zomars to provide his input on the same before going ahead and merging this. Blocking the merge until then 🙏

File: apps/web/contexts/contractsContext.tsx
Comment (hariombalhara) [reply apps/web/contexts/contractsContext.tsx:42]: Yeah, for this particular instance I have no idea why value is not assignable to Provider and why it is still not giving issues at runtime

File: packages/embeds/embed-core/src/preview.ts
Comment (hariombalhara) [reply packages/embeds/embed-core/src/preview.ts:11]: This is actually the embed snippet that we gave to users. This is already typed in the source. Didn't want to re-type it here.

**Review by zomars — APPROVED** [review]
awesome @hariombalhara my midnight brain didn't catch these 🙏🏽

**Pull request merged**
